<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompassOnboardCalibrationDialog</class>
 <widget class="QDialog" name="CompassOnboardCalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compass Onboard Calibration</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="panelVerticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Calibration Progress</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="graphHorizontalLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonBarVerticalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>248</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
